src/index.d.ts

```diff
@@ -506,6 +514,11 @@ export interface CardProps {
      * Add an image as the heading with the image prop
      */
     image?: ImageURISource;
+  
+    /**
+     * Optional properties to pass to the image if provided e.g "resizeMode"
+     */
+    imageProps?: ImageProperties;
```

Comment:
Why not just `Partial<ImageProperties>`. Why extend a new interface?

Reply:
Maybe some other components will/might need this partial `ImageProperties` it too? What do you think?

Reply:
Ah, I must be dreaming :) You're right -- this could lead some mistake when someone thinks that this new `ImageProperties` is come from `react-native` itself -- I'll fix it now. Thanks for the review.

Reply:
Sure thing 🙂